config/locales/en.yml

```diff
@@ -6,6 +6,28 @@ en:
       emails_found:
         one: "one email found"
         other: "%{count} emails found"
+  users:
+    delete_preview:
+      heading: "What do you want to do with your works?"
+      orphan: "orphan"
+      orphaning: "orphaning"
+      submit: "Save"
+      cancel: "Cancel"
+      confirm: "Are you sure? This can't be undone!"
+      options:
+        keep_pseud: "Leave my pseud but attach to the orphan account"
```

Comment:
I have to say (I know this is existing copy) that "keep pseud" and "leave my pseud" being used for the same action is twisting my mind into pretzels. I would be as confused as our users if I had to choose between these. 😬

Reply:
Yeah. I do not love the phrasing on this or the next one. Reworking the text on the various orphaning pages might be nice someday (although at least "Leave my pseud" is standard between this and the work page).